backend/store/instance_change_history.go

```diff
@@ -459,107 +459,36 @@ func (*Store) getLargestInstanceChangeHistorySequenceImpl(ctx context.Context, t
 
 	query := `
 		SELECT
-			MAX(sequence)
```

Comment:
Do we want MAX(sequence) or MAX(sequence)+1?

By the way, I think `COALESCE(MAX(sequence), 0)+1` looks more straightforward.

Reply:
This function is called getNextInstanceChangeHistorySequence() so the latter.

Good suggestion, changed to use COALESCE() instead.